scripts/deploy.js

```diff
@@ -0,0 +1,98 @@
+const { Web3Connection, ERC20, NetworkFactory, Network } = require("@taikai/dappkit");
+
+const argv = require('yargs')
+    .alias('e', 'envFile')
+    .describe('e', 'Environment file to load')
+    .default('e', '.env')
+    .alias('n', 'network')
+    .describe('n', 'Ethereum Blockchain')
+    .default('n', 'development')
```

Comment:
wouldn't it be better to have the user provide a rpc url directly, instead of relying on a hardcoded value that has to be mantaigned ?

Reply:
it think it would be great to have to move the networks variable to a file with a list of networks.

Reply:
Supports passed url and statically defined networks